src/App.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import plyr from 'plyr';
+import 'plyr/dist/plyr.css';
+import classes from './App.module.scss';
+
+const App = () => {
+  const [player, setPlayer] = React.useState(null);
+  const options = {
+    controls: [
+      'rewind',
+      'play',
+      'fast-forward',
+      'progress',
+      'current-time',
+      'duration',
+      'mute',
+      'volume',
+      'settings',
+      'fullscreen',
+    ],
+    // i18n: {
+    //     restart: "Restart",
+    //     rewind: "Rewind {seektime}s",
+    //     play: "Play",
+    //     pause: "Pause",
+    //     fastForward: "Forward {seektime}s",
+    //     seek: "Seek",
+    //     seekLabel: "{currentTime} of {duration}",
+    //     played: "Played",
+    //     buffered: "Buffered",
+    //     currentTime: "Current time",
+    //     duration: "Duration",
+    //     volume: "Volume",
+    //     mute: "Mute",
+    //     unmute: "Unmute",
+    //     enableCaptions: "Enable captions",
+    //     disableCaptions: "Disable captions",
+    //     download: "Download",
+    //     enterFullscreen: "Enter fullscreen",
+    //     exitFullscreen: "Exit fullscreen",
+    //     frameTitle: "Player for {title}",
+    //     captions: "Captions",
+    //     settings: "Settings",
+    //     menuBack: "Go back to previous menu",
+    //     speed: "Speed",
+    //     normal: "Normal",
+    //     quality: "Quality",
+    //     loop: "Loop"
+    // }
```

Comment:
Why we have commented this?

Reply:
Kept it for reference; what functions i could use on the player.